### DISCUSSION TITLE
[@types/react-lazy-load-image-component] onLoad prop missing

### DISCUSSION THREAD

**Discussion by Bygrilinho** [DefinitelyTyped/DefinitelyTyped#66447] [Issues with a @types package] [answered]
The `LazyLoadImage` component's `afterLoad` prop has been deprecated, and `onLoad` should be used instead.
There doesn't seem to be an `onLoad` prop in the type declaration right now.

https://github.com/Aljullu/react-lazy-load-image-component/commit/f3f92c7e344b522a928a0099a816b16c8442f106

[a comment by an automated account is not shown]

**Comment by danvk** [comment]
Fix in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66479

**Comment by danvk** [comment, marked as answer]
@Bygrilinho `@types/react-lazy-load-image-component@1.6.0` has the fix. Thanks for the suggestion!

**Reply by Bygrilinho** [reply]
Great, thank you!
